docs/interceptortypes.md

```diff
@@ -0,0 +1,56 @@
+<!--
+---
+linkTitle: "InterceptorType"
+weight: 9
+---
+-->
+# InterceptorType
+
+A `InterceptorType` is cluster scoped resource that registers a new Interceptor that 
```

Comment:
I'm not 100% sure we want this to be cluster scoped. In particular, I'm thinking about how we can expose custom interceptors that can only be used within a single tenant namespace.

Reply:
If its not cluster scoped, each new namespace would have to install their own version of Interceptors....could we not use RBAC to restrict if certain interceptors need to be limited?

Reply:
chatted with @wlynch about this. We will start off with a cluster scoped CRD but will implement a Namespace scoped version of this in the future as well.

Reply:
Thanks! Let's update the docs to reflect the change.